.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: DR2 Run Lambda Tests
```

Comment:
Are these still "Lambda" tests?

Reply:
Nope, copy and paste fun. I've changed it to DR2 Run Scalafmt as that's all we're doing.